ui/src/App.tsx

```diff
@@ -22,17 +22,15 @@ const Console = loadable(() => import('./app/console/Console'));
 const Login = loadable(() => import('./app/auth/Login'));
 const Settings = loadable(() => import('./app/Settings'));
 const Onboarding = loadable(() => import('./app/Onboarding'));
+const FirstTimeOnboarding = loadable(() => import('./app/FirstTimeOnboarding'));
 const Support = loadable(() => import('./app/Support'));
 const Preferences = loadable(() => import('./app/preferences/Preferences'));
 const Namespaces = loadable(() => import('./app/namespaces/Namespaces'));
 const Tokens = loadable(() => import('./app/tokens/Tokens'));
 
-const namespacesRoutes = [
+const namespacedRoutes = [
   {
-    element: <Flags />,
-    handle: {
-      namespaced: true
-    },
+    element: <FirstTimeOnboarding />,
```

Comment:
I would suggest to drop `FirstTimeOnboarding` and just replace it with
```suggestion
    element: <Onboarding firstTime={true} />,
    loader: () => {
      const state = store.getState();
      if (state?.user.completedOnboarding) {
        return redirect('/flags');
      }
      return null;
    }

Reply:
i like it